functions/components/metaTagInjector.ts

```diff
@@ -38,5 +42,9 @@ export class MetaTagInjector implements HTMLRewriterElementContentHandlers {
       this.append(element, 'twitter:image', this.input.image)
       this.append(element, 'twitter:image:alt', this.input.title)
     }
+
+    if (this.blockedPathsList) {
+      this.append(element, 'cf:blockedpaths', this.blockedPathsList)
+    }
```

Comment:
custom headers should begin with `x-`. I'd do the same with meta tags (`x:`) for consistency.

```suggestion
    const blockedPaths = request.headers.get('x-blocked-paths-list')
    if (blockedPaths) {
      this.append(element, 'x:blocked-paths', blockedPaths)
    }
```